azure_monitor/src/azure_monitor/options.py

```diff
@@ -49,6 +51,7 @@ def __init__(
         self,
         connection_string: str = None,
         instrumentation_key: str = None,
+        proxies: typing.Dict[str, str] = None,
```

Comment:
Maybe setting the default as {} instead of None to avoid check afterwards

Reply:
Mutable defaults are a common pitfall in Python. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments